apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs

```diff
@@ -14,22 +14,23 @@
     <main id="content" class="container-fluid">
       <%- include('../partials/courseSyncErrorsAndWarnings'); %>
       <% if (authz_data.has_course_permission_edit) { %>
-      <div class="modal fade" id="closeAllIssuesModal" tabindex="-1" role="dialog" aria-labelledby="closeAllIssuesModalLabel">
+      <div class="modal fade" id="closeMatchingIssuesModal" tabindex="-1" role="dialog" aria-labelledby="closeMatchingIssuesModalLabel">
         <div class="modal-dialog" role="document">
           <div class="modal-content">
             <div class="modal-header">
-              <h4 class="modal-title" id="closeAllIssuesModalLabel">Close all issues</h4>
+              <h4 class="modal-title" id="closeMatchingIssuesModalLabel">Close matching issues</h4>
             </div>
             <div class="modal-body">
-              Are you sure you want to close all issues? This cannot
-              be undone.
+              Are you sure you want to close <strong><%= rows.length %></strong> matching
```

Comment:
Should this be the count of open issues instead of all visible issues (which may have some closed ones)? Note that you already check for the existence of an open issue, might as well compute them and show a proper count.

Reply:
Done in d751cec.